test/database.test.js

```diff
@@ -0,0 +1,62 @@
+import {
+  afterAll,
+  beforeAll, beforeEach, describe, expect, it,
+} from 'vitest';
+import { client } from '../connections.js';
+import {
+  addNewSoldier, lookForSoldier, lookForAllSoldiers, dbName, dbCollection,
+} from '../database.js';
+
+beforeAll(async () => {
+  await client.connect();
+});
+
+afterAll(async () => {
+  await client.close();
+});
+
+beforeEach(async () => {
+  await client.db(dbName).collection(dbCollection).deleteMany({});
+});
+
+describe('addNewSoldier function', () => {
+  it('Should insert soldier', async () => {
```

Comment:
More like `it('Should return res with acknowledged true`

I am really pedantic about test description because it like comments... and lie comment are the devil 👿 
if you say "It Should insert soldier" check it!